build/makelib/golang.mk

```diff
@@ -162,12 +165,17 @@ go.fmt:
 
 go.validate: go.vet go.fmt
 
-.PHONY: go.vendor
-go.vendor $(GO_VENDOR_DIR)/vendor.stamp: $(GLIDE)
+$(GLIDE_LOCK): $(GLIDE) $(GLIDE_YAML)
```

Comment:
$(GLIDE_YAML) is a variable referring to the glide.yaml file. its defined above.